### DISCUSSION TITLE
How to Enable GridPostArchive?

### DISCUSSION THREAD

**Discussion by SL33pxxs** [flyntwp/flynt#506] [Q&A] [answered]
Hello, 
I Want create a page blog archive, but in the component not see a GridPostArchive ... 

Thanks in Advanced
![images](https://github.com/flyntwp/flynt/assets/28668567/0b2dc4d1-bedc-412d-97ef-d20bc9a17418)

**Comment by timohubois** [comment, marked as answer]
@SL33pxxs the component “GridPostsArchive” will be rendered inside the template [index.twig](https://github.com/flyntwp/flynt/blob/0ef7ec83dba205e1d9707c09c5bf9a8d5eae2f4d/templates/index.twig#L4).

To get it working you can create a new page and assign this page inside the reading settings as “Posts Page”.
<img width="772" alt="Screenshot 2023-08-15 at 22 46 20" src="https://github.com/flyntwp/flynt/assets/46505685/c0ddd669-02c4-42af-8cb6-5992cd33157b">

**Reply by SL33pxxs** [reply]
Whoo, Thanks! Works it
